### PR TITLE
Instantiate post method instead of the INSTANTIATE_POST action

### PR DESCRIPTION
**Pull request by maxme**
Instantiate post method instead of the INSTANTIATE_POST action

### CODE REVIEW

**Comment by maxme** [comment]
I need to fix cAT.

File: example/src/androidTest/java/org/wordpress/android/fluxc/release/ReleaseStack_PostTestXMLRPC.java [hunk 1]
```diff
@@ -840,11 +838,7 @@ public void testCreatePostAsSubscriber() throws InterruptedException {
         mNextEvent = TestEvents.POST_INSTANTIATED;
         mCountDownLatch = new CountDownLatch(1);
 
-        InstantiatePostPayload initPayload = new InstantiatePostPayload(subscriberSite, false);
-        mDispatcher.dispatch(PostActionBuilder.newInstantiatePostAction(initPayload));
-
-        assertTrue(mCountDownLatch.await(TestUtils.DEFAULT_TIMEOUT_MS, TimeUnit.MILLISECONDS));
-
+        createNewPost();
```
Comment (aforcier) [inline example/src/androidTest/java/org/wordpress/android/fluxc/release/ReleaseStack_PostTestXMLRPC.java]: This breaks `testCreatePostAsSubscriber()`, we need to specifically create a post for `subscriberSite`.

**Comment by aforcier** [comment]
This looks good except for the broken test 👍.

(I'll wait for a WPAndroid-side PR before merging this.)

**Comment by aforcier** [comment]
:shipit:

**Pull request merged**